src/main/resources/servers/broad-dev.json

```diff
@@ -5,5 +5,6 @@
   "dataRepoUri": "https://jade.datarepo-dev.broadinstitute.org",
   "samUri": "https://sam.dsde-dev.broadinstitute.org",
   "workspaceManagerUri": "https://workspace.dsde-dev.broadinstitute.org",
-  "wsmDefaultSpendProfile": "wm-default-spend-profile"
+  "wsmDefaultSpendProfile": "wm-default-spend-profile",
+  "useJanitorForExternalResourcesCreatedByTests": true
```

Comment:
iiuc the current behavior is that this is on for all environments. should it also be enabled for other Broad servers to match the current state?

Reply:
Done